Frontend/src/components/Input/PopupFormular.svelte

```diff
@@ -124,7 +126,18 @@
               );
             }
           });
-        dispatch("save");
+          let emptyInputs = popupFormularConfiguration.inputs.filter((input) => {
+            const value = String(
+              $keyValueStore[input.bindTo.keyValueStoreKey][input.bindTo.attr]
+            ).trim();
+            return value.length === 0 && input.required !== false;
+          });
+          if (emptyInputs.length > 0){
+            console.log(emptyInputs);
+            dispatch("error", {"emptyInputs": emptyInputs, "emptyInputLabels": emptyInputs.map((input) => input.label)});
+          } else {
+            dispatch("save");
```

Comment:
I think we should still allow the user to save and just show the warning if there are emptyInputs.

Reply:
good point